src/page/search/common/component/results.js

```diff
@@ -2,9 +2,18 @@
 import React from 'react';
 import { translate } from 'focus-core/translation';
 import builder from 'focus-core/component/builder';
-import { assign, mapValues, keys, omit } from 'lodash/object';
-import { clone, isArray } from 'lodash/lang';
-import { filter, map, groupBy } from 'lodash/collection';
+
+import assign from 'lodash/object/assign';
```

Comment:
should use js' assign function

Reply:
True. We should also get rid of object-assign and clean it